app/services/feeds/import.rb

```diff
@@ -29,8 +29,8 @@ def call
         # NOTE: doing this sequentially to avoid locking problems with the DB
         # and unnecessary conflicts
         articles = feedjira_objects.flat_map do |user_id, feed|
-          # TODO: replace `feed` with `feed.url` as `RssReader::Assembler`
-          # only actually needs feed.url
+          # TODO: replace `feed` with `feed.url` as `Feeds::AssembleArticleMarkdown`
+          # only actually uses feed.url
```

Comment:
not going to optimize this right now, will leave for a future downtime as it requires a bit of shuffling that's not worth it right now. We can address this when we optimize `Feeds::AssembleArticleMarkdown`